MapboxVision/VisionManager/VisionReplayManager.swift

```diff
@@ -12,8 +12,9 @@ import CoreMedia
 /**
     `VisionReplayManager` is a counterpart of `VisionManager` that uses recorded video and telemetry instead of realtime data.
     Use it in the same workflow as you use `VisionManager` after creating it with specific recorded session.
-*/
 
+    - Important: This class is intended for debugging purposes only.
```

Comment:
I'd recommend to add `Do NOT use session recording in production applications.` explicitly as well.

Reply:
Please propose in mapbox/mapbox-vision#1042. This is just a mirror.
Concerning particular thing seems like this is the same thing with different words.